circuit/types/string/src/helpers/to_bits.rs

```diff
@@ -58,7 +59,11 @@ mod tests {
 
         for i in 0..ITERATIONS {
             // Sample a random string. Take 1/4th to ensure we fit for all code points.
-            let expected: String = (0..(Circuit::MAX_STRING_BYTES - i) / 4).map(|_| rng.gen::<char>()).collect();
+            let expected: String = (0..(Circuit::MAX_STRING_BYTES - i) / 4)
```

Comment:
this doesn't come from this PR, but is there a reason why we subtract `i` in this one case?

Reply:
I've added a boolean toggle for whether to randomly sample a fixed-length or variable-length string